package.json

```diff
@@ -47,8 +47,9 @@
     "test:storybook:debug": "PWDEBUG=1 pnpm test:storybook:local",
     "test:storybook:gen": "playwright codegen localhost:54000/",
     "types": "vue-tsc --noEmit",
-    "lint": "prettier --check .; eslint --ext .js,.vue,.ts --ignore-path .gitignore --ignore-path .eslintignore --max-warnings=0 .",
-    "lint:fix": "prettier --write --loglevel=warn .; pnpm lint --fix",
+    "pc:download": "curl https://api.github.com/repos/pre-commit/pre-commit/releases/latest | grep -o 'https://.*\\.pyz' -m 1 | xargs | xargs curl -fsJo pre-commit.pyz -L",
+    "pc:install": "python3 pre-commit.pyz install -t pre-push -t pre-commit",
+    "pc:lint": "python3 pre-commit.pyz run --all-files",
```

Comment:
This is such a smart way of making this simple for a non-python repository without polluting the global Python installation. Kudos, Dhruv :rocket: